backend/typescript/services/implementations/authService.ts

```diff
@@ -177,26 +180,28 @@ class AuthService implements IAuthService {
     }
 
     try {
-      const passwordResetLink = await firebaseAdmin
-        .auth()
-        .generatePasswordResetLink(email);
-      // first-time determines if we are setting a new account password
-      // (account will be verified) or reseting an old account's password
-      const setPasswordLink = passwordResetLink.concat("&first-time=true");
+      const authId = await this.userService.getAuthIdById(user.id);
+
+      const setPasswordLink = `${process.env.CLIENT_URL}/auth/action?mode=verify-user&uid=${authId}`;
 
       const emailBody = `
```

Comment:
> Should we be defining these in some constants/content file rather than inline?

We probably should but I think that would be a different ticket since it would change not just this but other things too probably